continue.css

```diff
@@ -1,300 +0,0 @@
-/* general body CSS below --------------------------------------*/
-body {
-    background-color: #FFFAF7;
-}
-
-hr{
-    margin: 0.25em 0px;
-}
-
-h4{
-    font-size: 10px;
-}
-
-h2 {
-    font-family: "Open Sans Condensed";
-    color: #4B3021;
-    font-size: 200%;
-    text-transform: uppercase;   
-}
-
-h3{
-    font-weight: 600;
-    font-family: 'Open Sans Condensed';
-    color: #4B3021;
-    text-transform: uppercase;
-}
-p {
-    font-family: 'Raleway', light;
-    font-weight: 150;
-    font-size: 100%;
-    text-align: justify;
-    color: #86220A;
-    margin-bottom: 10px;
-}
-
-/* header CSS below --------------------------------------------*/
-#header {
-    text-align: center;
-}
-
-#header img {
-    text-align: center;
-    width: 30%;
-    height: auto;
-}
-   
-
-/* main content CSS below-------------------------------------------*/
-
-.main-contents{
-    display: grid;
-    grid-gap: 2rem;
-    grid-template-columns:  10% 55% 25% 10%;
-    grid-template-rows: 10% 45% 30% 15%;
-    grid-template-areas: 
-    " . header header ."
-    " . main-paragraph sidebar . "
-    " . main-paragraph sidebar . "
-    " footer footer footer footer"
-    }
-#header{
-    grid-area: header
-}
-#main-paragraph{
-    grid-area: main-paragraph;
-    margin-right: 75px;
-      }
-#main-paragraph > img {
-   
-   justify-content: stretch;
-}
-#sidebar{
-    grid-area: sidebar
-}
-#footer{
-    grid-area: footer
-}
-
-/* button CSS Below ---------------------------------------*/
-
-.learnmore-button {
-    background-color: #8F4307;
-    color: #FDE6BE;
-    font-size: 1rem;
-    border: 0;
-    padding: 0 20px;
-    text-align: center;
-    border-radius: 5px;
-    line-height: 2.5;
-}
-.learnmore-button:hover {
-    background-color: rgb(161, 83, 30);
-}
-
-.reading-button {
-    background-color: #C45C09;
-    color: #FFBC0A;
-    font-size: 1rem;
-    border: 0;
-    padding: 0 20px;
-    text-align: center;
-    border-radius: 5px;
-    line-height: 2.5;
-}
-
-.reading-button:hover {
-    background-color: rgb(161, 83, 30)
-}
-.date-label {
-    background-color: #764248;
-    border-radius: 6px;
-    color: #FDE6BE;
-    padding: .25rem;
-    width: 25%;
-    height: auto;
-    box-shadow: darkgrey;
-    font-family: "Raleway", light;
-}
-.date-label:hover {
-    background-color: #61373B;
-}
-
-/* footer CSS Below ---------------------------------*/
-#footer {
-    margin: 1em 0 0 0;
-    padding: 1em;
-    background-color: rgb(145, 162, 168);
-    border-top: 1px solid rgb(122, 117, 117);
-    box-shadow: 0 50vh 0 50vh rgb(145, 162, 168);
-}
-h2#footer-title {
-    text-align: center;
-    margin: 5vh auto 3vh auto;
-    font-size: 1em;
-    letter-spacing: 5px;
-    color: black;
-    font-weight: lighter;
-    }
-div#form-space {
-        display: flex;
-        justify-content: space-evenly;
-      }
```

Comment:
Format your code, and add a line of whitespace between different css rulesets. It doesn't matter to the computer, but it makes your code much easier to read.